libnavigation-util/src/main/java/com/mapbox/common/NativeLoggerWrapper.kt

```diff
@@ -20,4 +20,6 @@ internal object NativeLoggerWrapper {
     fun error(message: String, category: String?) {
         Log.error(message, category)
     }
+
+    fun getLogLevel(category: String) = LogConfiguration.getLoggingLevel(category)
```

Comment:
Is this check enough?

What happens if we do:
```
LogConfiguration.setLoggingLevel(ERROR)
LogConfiguration.setLoggingLevel("nav-sdk", INFO)
```
and the do
```
LogConfiguration.getLoggingLevel(category)
```

Will this return `ERROR` or `INFO`? In practice though, does the global log level overwrite the category level, or the other way around? That's something we'll need to adjust for in our checks.

Reply:
<img width="834" alt="image" src="https://user-images.githubusercontent.com/6190346/194301326-c7fac10d-266f-48d7-a38f-80c8dcdb8f87.png">

According to documentation it should return `INFO`. 


<img width="1209" alt="image" src="https://user-images.githubusercontent.com/6190346/194302033-d552ef1d-a9a5-4b21-b125-3ba92f2d81ae.png">

Reply:
Thanks for double-checking, I think a doc update is enough https://github.com/mapbox/mapbox-sdk-common/pull/3130.